README.md

```diff
@@ -1,58 +1,69 @@
-# node-jwt-simple
+<div align="center">
+    <img src="https://jwt.io/assets/logo.svg"><br><br>
+</div>
 
-[JWT(JSON Web Token)](http://self-issued.info/docs/draft-jones-json-web-token.html) encode and decode module for node.js.
+# Welcome
+Secure your application in seconds using standardised **JWT-Tokens**. [RFC 7519](https://tools.ietf.org/html/rfc7519)
 
-## Install
+[![npm version](https://badge.fury.io/js/jwt-simple.svg)](https://badge.fury.io/js/jwt-simple)
 
+## Sample Use-Case
+You may use this as an authentication-method for your API: on one route you encode a token, storing the authenticated user (e.g. after checking password and email) in it. On all other routes you check if the token is valid - this indicates that the user once did authenticate himself successfully.
+
+## Installation
     $ npm install jwt-simple
 
 ## Usage
+### Encoding
+Encode a Object into your token - the `jwt.encode` function takes a maximum of 3 parameters:
```

Comment:
Using the signature of the method in the title allows the user to get an idea of the API right from the start, even before reading the doc, which is good !